models/autoencoders/dvaepp.py

```diff
@@ -0,0 +1,121 @@
+"""
```

Comment:
I changed this file slightly wrt your original version to make it compatible with the refactoring. Let me know if there are questions or issues.